app/views/shared/_sidebar.html.erb

```diff
@@ -40,6 +40,9 @@
           <li>
             <%= link_to "Videos", "/videos", sidebar_link_options("videos") %>
           </li>
+          <li>
+            <%= link_to "Credential helpers", "/doc/credential-helpers", sidebar_link_options("credential-helpers") %>
+          </li>
```

Comment:
I could go either way on this; I don't think that we necessarily need to have a link to this new page permanently in the sidebar. Maybe something like this instead:

```diff
diff --git a/app/views/shared/_sidebar.html.erb b/app/views/shared/_sidebar.html.erb
index 440b1265..d10c874b 100644
--- a/app/views/shared/_sidebar.html.erb
+++ b/app/views/shared/_sidebar.html.erb
@@ -40,9 +40,6 @@
           <li>
             <%= link_to "Videos", "/videos", sidebar_link_options("videos") %>
           </li>
-          <li>
-            <%= link_to "Credential helpers", "/doc/credential-helpers", sidebar_link_options("credential-helpers") %>
-          </li>
           <li>
             <%= link_to "External Links", "/doc/ext", sidebar_link_options("external-links") %>
           </li>
diff --git a/app/views/shared/ref/_external.html.erb b/app/views/shared/ref/_external.html.erb
index bd826058..970c36f5 100644
--- a/app/views/shared/ref/_external.html.erb
+++ b/app/views/shared/ref/_external.html.erb
@@ -2,4 +2,5 @@
 <ul class='unstyled'>
   <li><%= man('git-svn') %></li>
   <li><%= man('git-fast-import') %></li>
+  <li><%= link_to "Credential helpers", "/doc/credential-helpers" %></li>
 </ul>
```

Reply:
In that sense, shouldn't this be part of Git's manual pages instead of git-scm.com? `man git-credential-helpers`, or `man git-credential-helper-faq` both sound reasonable to me (and have some precedence in the form of `man gitfaq`, etc.)

Reply:
It does seem a bit specific to surface in the sidebar. However, I could see it possibly going under **Setup and Config** (which is at the top of the **Reference** list and easier to discover imo) rather than **External Systems**:

![Screenshot 2023-06-07 at 5 21 09 PM](https://github.com/git/git-scm.com/assets/11321782/c61cf066-4512-479e-ab48-91f7c0b5e6d3)

![Screenshot 2023-06-07 at 5 22 27 PM](https://github.com/git/git-scm.com/assets/11321782/fd3ff817-8d38-440a-96aa-cf854d4b4608)